docs/modules/ROOT/pages/hazelcast-platform-operator-external-backup-restore.adoc

```diff
@@ -0,0 +1,500 @@
+= External Backup and Restore with Hazelcast
```

Comment:
```suggestion
= Restore a Cluster from Cloud Storage with Hazelcast Platform Operator
```

The title should reflect the goal rather than the features we use.